### PR TITLE
List menus

### PR DESCRIPTION
**Pull request by MuhammadHamza096**
### Goal
food items now shows in cards

### Work
create card and tab component from material ui responsive 

### Risks
N/A

### Trello Link
https://trello.com/c/qgxsRN2L/10-design-a-card-for-displaying-meal-with-its-description-along-with-price

### CODE REVIEW

File: src/Mock/Data.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import curry from '../components/images/curry.jpg';
```
Comment (SiddiquiMonis401) [inline src/Mock/Data.js:1]: Images are consider to be an asset for the app so please make a directory called assets and keep images there ..

File: src/components/Button/index.js [hunk 2]
```diff
@@ -1,16 +1,15 @@
...
 import { Button as MuiButton, Typography } from '@material-ui/core';
 
 export default function Button(props) {
   // One option to get theme and supply it to styled compoenents
   return (
     <MuiButton color="primary" variant="contained">
-      <Typography variant="button">
-        Order Food
-      </Typography>
+      <Typography variant="button">Order Food</Typography>
```
Comment (SiddiquiMonis401) [inline src/components/Button/index.js:9]: I think if it is a common component we must use props to display the title of the button ..

File: src/components/CardMenus/CommonCard.js [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+  return (
+    <div>
+      <Card className={classes.root}>
+        <Box className={classes.image}>
+          <CardMedia className={classes.media} image={img} title={name} />
+        </Box>
+        <CardHeader
+          className={classes.header}
+          // eslint-disable-next-line react/jsx-no-duplicate-props
```
Comment (SiddiquiMonis401) [inline src/components/CardMenus/CommonCard.js:63]: Why are you using eslint disable for duplicate props when there isn't a duplicate prop.

File: src/components/CardMenus/Tabs.js [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+    <>
+      <Paper square>
+        <Tabs
+          aria-label="disabled tabs example"
+          indicatorColor="primary"
+          onChange={handleChange}
+          textColor="primary"
+          value={value}
+        >
+          <Tab label="Active" />
```
Comment (SiddiquiMonis401) [inline src/components/CardMenus/Tabs.js:21]: What that same label "Active" n times means ... are u just render Active for all ?? If yes, then please use some appropriate name ..

**Review by SiddiquiMonis401 — CHANGES_REQUESTED** [review]

**Review by SiddiquiMonis401 — APPROVED** [review]

**Review by ShahirZain — APPROVED** [review]

**Pull request merged**